test_scripts/Polices/appID_Management/ATF_P_TC_OnAppPermissionChanged_appRevoked_true.lua

```diff
@@ -0,0 +1,70 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+--     [ChangeRegistration]: appRevoked:true
+-- Description:
+-- <appID> application is assigned with specific policies (for example, appID = 123abc->
+-- "app_policies" -> "123abc" : { <specific policies> }
+-- Steps:
+-- 1. any PolicyTableUpdate trigger happens
+-- 2. PTU is valid -> <appID> gets "null" policy
+-- Expected:
+-- SDL -> HMI: OnAppPermissionChanged (<appID>, appRevoked=true, params)
+--
+-- Test:
+--     1. Preconditions:
+--        Start SDL
+--
+--     2. Steps
```

Comment:
Next section is 
2. Performed steps 
---where you describe what steps will be executed in script in this case:
1. Any PolicyTableUpdate trigger happens
2. PTU is valid -> <appID> gets "null" polciies